### ISSUE TITLE
Use query substitution

### ISSUE THREAD

**Issue by kwannoel** [Holmusk/HolmuskDB#42]
See here https://github.com/Holmusk/HolmuskDB/pull/40#discussion_r435066611 for discussion.

- [x] Refactor `convertAddPerson` and sub-functions to include this

https://neo4j.com/developer/javascript/ for implementation example

**Comment by kwannoel** [comment]
Query substitution only works for `terms`:

```
CREATE (a:Person {name: $name}) RETURN a)
{ name: "James" }
```

For LABELS and RELATIONSHIPS, substitution this way is invalid

```
CREATE (a:$person ... )
{ person: "PERSON" }
```

We can rely on a validation layer which ensures labels and relations are
1) alphanumeric
2) If there are spaces, replace them with underscores
3) Otherwise reject

That way we avoid we can avoid injection.

**Comment by kwannoel** [comment]
Closed by #40
